### PR TITLE
Add strings for the Premium waitlist page

### PR DESCRIPTION
**Pull request by Vinnl**
The strings can be seen in action here: https://deploy-preview-1846--fx-relay-demo.netlify.app/premium/waitlist/

### CODE REVIEW

File: en/app.ftl [hunk 1]
```diff
@@ -330,6 +330,22 @@ premium-promo-use-cases-gaming-body-2 = Worried about other gamers or trolls lea
 
 premium-promo-pricing-free-price = Your current Free plan
 
+## The Premium waitlist page
+
+waitlist-heading = Join the { -brand-name-relay-premium } Waitlist
+waitlist-lead = Get notified when { -brand-name-firefox-relay-premium } is available for your region.
+waitlist-control-required = Required
+waitlist-control-email-label = What is your email address?
+waitlist-control-email-placeholder = yourname@example.com
```
Comment (peiying2) [inline en/app.ftl:339]: Does it matter whether this is translated or not? If it is, let's add a comment.
See examples from mozilla.org: https://transvision.mozfr.org/?recherche=yourname%40example.com&repo=mozilla_org&sourcelocale=en&locale=de&search_type=strings_entities

Comment (Vinnl) [reply en/app.ftl:339]: I'm not actually sure what the best practice is here? The main thing to demonstrate is that it takes an email address, so anything of that general form should work, but I can imagine that localisation is still useful - with the caveat that example.com is an actual example domain, for which I don't think there are local equivalents?

**Comment by Vinnl** [comment]
Just added a couple more strings for where we'll be linking to the waitlist page: f0b6707

File: en/app.ftl [hunk 2]
```diff
@@ -215,7 +215,12 @@ landing-pricing-free-price = Free
 landing-pricing-free-feature-1 = Up to 5 email aliases
 landing-pricing-free-feature-1-2 = Up to 5 email masks
 landing-pricing-free-feature-2 = Browser extension
+landing-pricing-free-feature-free = Free
+landing-pricing-free-description = Try { -brand-name-firefox-relay } aliases and start protecting your email inbox.
```
Comment (flodolo) [inline en/app.ftl:219]: masks?

Also, when are we planning to clean up the old "alias" strings, to avoid confusion?

Comment (Vinnl) [reply en/app.ftl]: Aaargh! Good eye: ac4cbad

And good point, I'll create a separate PR for that right away, as the content refresh went live the day before yesterday.

File: en/app.ftl (hunk 1 above)
Comment (flodolo) [reply en/app.ftl:347]: Please add a comment explaining that example.com should not be translated, only `yourname`. As you said, that's a IANA domain that is safe to use, unlike translated ones (there are only a couple of exceptions that I'm aware of, but not worth calling out).

File: en/app.ftl [hunk 3]
```diff
@@ -330,6 +338,22 @@ premium-promo-use-cases-gaming-body-2 = Worried about other gamers or trolls lea
...
+
+waitlist-heading = Join the { -brand-name-relay-premium } Waitlist
+waitlist-lead = Get notified when { -brand-name-firefox-relay-premium } is available for your region.
+waitlist-control-required = Required
+waitlist-control-email-label = What is your email address?
+waitlist-control-email-placeholder = yourname@example.com
+waitlist-control-country-label = What country do you live in?
+waitlist-control-locale-label = Select your preferred language.
+waitlist-submit-label = Join the Waitlist
+waitlist-privacy-policy-agree = By clicking “Join the waitlist”, you agree to our <a>Privacy Policy</a>.
```
Comment (flodolo) [inline en/app.ftl:351]: Nit: the label has Waitlist uppercase, assuming it's the one above?

**Review by flodolo — APPROVED** [review]

File: en/app.ftl (hunk 3 above)
Comment (Vinnl) [reply en/app.ftl:351]: Wow, you really do have a sharp eye! :)

image

What do you think of the approach in 4f64596, where I just pull the correct value from a variable?

File: en/app.ftl (hunk 1 above)
Comment (Vinnl) [reply en/app.ftl:348]: Done in 5c3ee31.

File: en/app.ftl [hunk 4]
```diff
@@ -330,6 +338,25 @@ premium-promo-use-cases-gaming-body-2 = Worried about other gamers or trolls lea
...
+waitlist-control-required = Required
+waitlist-control-email-label = What is your email address?
+# Please only translate `yourname`; example.com is an actual example domain that is safe to use.
+waitlist-control-email-placeholder = yourname@example.com
+waitlist-control-country-label = What country do you live in?
+waitlist-control-locale-label = Select your preferred language.
+waitlist-submit-label = Join the Waitlist
+# Variables:
+#  $joinButtonLabel (string): the value of waitlist-submit-label
+waitlist-privacy-policy-agree = By clicking “{ $joinButtonLabel }”, you agree to our <a>Privacy Policy</a>.
```
Comment (flodolo) [inline en/app.ftl:354]: You should be able to use an actual message reference here? Please verify that it works as expected though.

```suggestion
waitlist-privacy-policy-agree = By clicking “{ waitlist-submit-label }”, you agree to our <a>Privacy Policy</a>.
```

Comment (Vinnl) [reply en/app.ftl]: Ha, of course I can, that makes much more sense! It does work as expected, thanks :)

**Review by flodolo — APPROVED** [review]

**Review by peiying2 — APPROVED** [review]

**Comment by peiying2** [comment]
flod has owl eyes!

File: en/app.ftl [hunk 5]
```diff
@@ -1088,3 +1115,7 @@ forwarded-email-footer = Stop email forwarding and manage settings for all alias
 forwarded-email-footer-2 = Stop email forwarding and manage settings for all masks here.
 # This entire text is a link
 forwarded-email-footer-premium-banner = Upgrade to { -brand-name-relay-premium }
+
+## Notifications component
+
+toast-button-close-label = Close notification
```
Comment (flodolo) [inline en/app.ftl:1121]: @lloan 

Adding strings after the PR was reviewed and then merging right away is not OK, it defies the point of having mandatory reviews.

In particular, this string should have landed leaving an empty line at the end of the file. Next time someone adds a string after, this one will change :-|

**Pull request merged**